services/movie_session.py

```diff
@@ -0,0 +1,40 @@
+from datetime import datetime
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                cinema_hall_id=cinema_hall_id,
+                                movie_id=movie_id)
+
+
+def get_movies_sessions(session_date: str = None) -> MovieSession:
+    session = MovieSession.objects.all()
+    if session_date:
+        return session.filter(show_time__date=session_date)
```

Comment:
You are using the `.filter()` method to retrieve objects. However, the `.filter()` method returns a `QuerySet` of objects, not a single object. This contradicts with the return type in the function signature. If you want to get multiple objects, use `.filter()`, but correct the return type in the function signature.